benchmarks/experiment_runner.py

```diff
@@ -453,6 +486,20 @@ def parse_args(args=None):
       help="JSON string of the model config dict.",
   )
 
+  parser.add_argument(
+      "--profile-cuda",
```

Comment:
When does it make sense to pass --profile-cuda and not pass --profile-cuda-dump? If it doesn't, we should merge the two flags.

Reply:
Check the desc. I've a PR ready which introduces a flag which will allow for gathering CPU + CUDA time, so you can either use profile-cuda-dump, or new flag or both. Merging profiling instantiation into either of flags, or both complicates a code (as you have to check whether profiler was instantiated already etc.), so that's why it's separated.

Because it's git, and it's hard to do the stacked diffs I'm keeping it locally.

Reply:
Missed that, thanks! I am new to this reviewing UI -- apologies.
Re: stacked diffs, git can help a bit with --update-refs: https://andrewlock.net/working-with-stacked-branches-in-git-is-easier-with-update-refs/
Note: AFAIK on Github in order to show the right diff's, one has to push all the branches involved to the repo in question -- i.e. one cannot choose a branch in a fork as a diff base. Perhaps that's why this repo has 2000+ branches?